app/src/main/java/org/schabi/newpipe/error/ErrorPanelHelper.kt

```diff
@@ -69,19 +73,17 @@ class ErrorPanelHelper(
         errorServiceExplanationTextView.isVisible = false
         errorActionButton.isVisible = false
         errorRetryButton.isVisible = false
+        errorOpenbrowserButton.isVisible = false
     }
 
     fun showError(errorInfo: ErrorInfo) {
-
```

Comment:
Same as above.

Reply:
I didn't change this line of code either, it was already there, are you sure I want to remove it? Also if I delete it, the file will error out

Reply:
See my last comment above.